sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestBase.java

```diff
@@ -67,7 +68,7 @@
 /**
  * Test base for resource manager SDK.
  */
-public abstract class ResourceManagerTestBase extends TestBase {
+public abstract class ResourceManagerTestBase extends TestProxyTestBase {
```

Comment:
We might need another `ResourceManagerProxyTestBase` when you finalize. We likely won't migrate all 20+ lib at the same time.